man/systemd.network.xml

```diff
@@ -2287,6 +2287,56 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
```

Comment:
I think you're missing a "parent" somewhere in the first sentence.

Reply:
Where the parent would be applied and how root is different ?

Reply:
I'm only suggesting to change the first sentence to "Specifies the parent Queueing Discipline (qdisc)."

A future enhancement would be to support attaching a qdisc as a child of another qdisc.